kibana/values.yaml

```diff
@@ -21,23 +30,23 @@ secretMounts: []
 #    path: /usr/share/kibana/data/kibana.keystore
 #    subPath: kibana.keystore # optional
 
-image: "docker.elastic.co/kibana/kibana"
-imageTag: "7.3.0"
-imagePullPolicy: "IfNotPresent"
-
 # additionals labels
 labels: {}
 
 podAnnotations: {}
   # iam.amazonaws.com/role: es-cluster
 
-resources:
-  requests:
-    cpu: "100m"
-    memory: "500m"
-  limits:
-    cpu: "1000m"
-    memory: "1Gi"
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
```

Comment:
This one is also a breaking change. I do not agree with the default helm chart templates decision to leave these blank. Leaving the resources empty puts the pods into a "best effort class" where Kubernetes will happily kill them to make room for something else which does have them set. If a user really wants their pods to be best effort they can set this to `{}`. However out of the box it is important to make sure that a user installing with the default settings has the best possible experience and that includes having a stable environment. I would rather someone fails to deploy Kibana because they don't have enough memory resources, versus someone having their containers stopped in production because the default resources weren't being set. 

Sacrificing availability to make it easy for environments like Minikube isn't what we want to optimize the defaults for. Instead we have [examples](https://github.com/elastic/helm-charts/tree/master/elasticsearch#minikube) (configured with low memory requirements) specifically for running in Minikube as there are some other production ready defaults which prevent the charts from working on a single node in a low resource environment.